### ISSUE TITLE
delegate seems not to work

### ISSUE THREAD

**Issue by stratboy** [atomiks/tippyjs#728]
Hi, the supposed bug is about delegate, so I cannot reproduce in codepen.

I'm currently working on a page which contains code like this:

`<div class="component-screws"></div>`

I want to delegate like this:

```
      const component_screws_tips = delegate('.component-screws', {
        target: '.component-show-screw-details'
        ,trigger: 'click'
        ,content: 'ok'
      });
```

It won't work. Meaning that, if `.component-show-screw-details` elements are dynamically added later, they won't trigger any tippy. 

It does work if elements are already there:

```
<div class="component-screws">
    <div class=".component-show-screw-details">ok</div>
</div>
```

But that makes delegation useless.

**Comment by stratboy** [comment]
FINDING #1: if, at instantiation time, there's at least 1 child target inside the parent, then delegation will work on later added child elements.

In other words, it seems delegation won't work only if initially there are no children.

**Comment by stratboy** [comment]
FINDING #2: offtopic but should be taken into consideration > even when it works, I think there a problem on the content function implementation. it gets passed only the parent element, which is not so usefull in this use case. Should also be passed the target element.

```
        const component_screws_tips = delegate('.component-screws', {
          target: '.component-show-screw-details'
          ,trigger: 'click'
          ,content(parent){ // <<< should be passed also the target element
            return 'test';
          }
        });
```

**Comment by atomiks** [comment]
I can't reproduce the first one (you can use `tippy.delegate()` in the CodePen btw). The second one is a problem as it should use the target element like `appendTo` does... I think v5 was the same tho, usually you'll use `data-tippy-content` with event delegation however

**Comment by atomiks** [comment]
Actually can't reproduce either... https://codepen.io/atomiks/pen/JjdOWvB

**Comment by stratboy** [comment]
I found out my issue > parent wasn't there on instantiation. 

Issue #2 is ok too, I solved by upgrading to v6 (was 5.2.0) and now the item passed to content() is the real target.

Thank you
